examples/02-registered-shopper-auth-token.ts

```diff
@@ -8,74 +8,52 @@
 /**
  * Get authorization token for a registered customer
  * Usage: ts-node examples/02-registered-shopper-auth-token.ts
- * Note: Replace configuration parameters and user credentials before running
  */
-import { ClientConfig, Customer } from "../dist";
-import {
-  getObjectFromResponse,
-  ResponseError,
-  ShopperToken,
-  stripBearer,
-} from "@commerce-apps/core";
-import ShopperCustomers = Customer.ShopperCustomers;
+import { ClientConfig, Customer } from "commerce-sdk";
 
-//client configuration parameters
+// client credentials
```

Comment:
```suggestion
// client credentials
```
Maybe add a note that these are demo credentials, and that they can/should substitute their own? Not sure of the best phrasing.

Reply:
Default credentials maybe?